draft-ietf-quic-ack-frequency.md

```diff
@@ -340,6 +340,15 @@ see {{out-of-order}}.
 
 {{congestion}} and {{batch}} describe exceptions to this strategy.
 
+The requirement that all packets have to be acknowleged at least once,
+as stated in {{Section 13.2.1 of QUIC-TRANSPORT}} is not affected by this
+extension. However, as discussed in {{Section 13.2.4 of QUIC-TRANSPORT}} this does not
+guarantee that every acknowledgment is seen by the sender before it is no
+longer included in the ACK frame. Therefore when a larger ack delay is used,
+the effect of lost or re-ordered packets with ACK frames must be considered
+carefully when triming the ACK range.
```

Comment:
```suggestion
extension. Unlike {{QUIC-TRANSPORT}}, however, large values for
reordering_threshold and ack_eliciting threshold can prevent an
ACK frame from being sent although there are multiple ACK ranges.
Implementations need to be certain to send an ACK before any
unsent ACK range is truncated.
```

Reply:
Ok, my text can be added separately to fix my issue.

Having an ACK range has been a trigger in RFC9000, because any OOO packet is an ACK trigger.  That's why an ACK_FREQUENCY implementer probably has to write new code for this case. I'm open to different wording, but the reason I'm energized about this issue is because this is, in fact, new code subtly required by this draft.

Reply:
Okay, then let's try to add a separate note. Do you want to make another proposal, or should I try to adapt your text?